src/content/changes/highlights/2021-05-21.md

```diff
@@ -11,7 +11,7 @@ owner:
 
 [Cloudflared v0.1.0](https://docs.giantswarm.io/changes/managed-apps/cloudflared-app/v0.1.0/) is now available. The app allows you to launch Cloudflare Argo Tunnels that route directly to services inside your cluster. Useful for clusters not running in a public cloud provider. This release significantly eases the installation in Kubernetes by creating a single deployment that reuses a named tunnel.
 
-[Prometheus Operator v0.8.2](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.2/) upgrades prometheus to [v2.26.1](https://github.com/prometheus/prometheus/releases/tag/v2.26.1) which contains a fix for [CVE-2021-29622](https://github.com/prometheus/prometheus/security/advisories/GHSA-vx57-7f4q-fpc7). The vulnerability allowed attackers to redirect to arbitrary URLs through a specially crafted address (e.g.: http://127.0.0.1:9090/new/newurl). Prior to that, [v0.8.1](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.1/) eases installing on kubernetes >= 1.18 by enabling `ingressClass` and `pathType` by default.
+[Prometheus Operator v0.8.2](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.2/) upgrades prometheus to [v2.26.1](https://github.com/prometheus/prometheus/releases/tag/v2.26.1) which contains a fix for [CVE-2021-29622](https://github.com/prometheus/prometheus/security/advisories/GHSA-vx57-7f4q-fpc7). The vulnerability allowed attackers to redirect to arbitrary URLs through a specially crafted address (e.g.: https://127.0.0.1:9090/new/newurl). Prior to that, [v0.8.1](https://docs.giantswarm.io/changes/managed-apps/prometheus-operator-app/v0.8.1/) eases installing on kubernetes >= 1.18 by enabling `ingressClass` and `pathType` by default.
```

Comment:
Not a big deal but this one was actually the example text provided in the the CVE description

Reply:
yeah good one